tests/driver_cst816s/main.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the CST816S touch screen driver
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ *
+ * @}
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "cst816s_params.h"
+#include "cst816s.h"
+#include "xtimer.h"
+#include "fmt.h"
```

Comment:
AFAICS, nor xtimer or fmt are used in this app

Reply:
Removed!